bootstrap-into-dart.md

```diff
@@ -19,34 +19,33 @@ We looked at a lot of languages, and
 We hope these resources make Dart
 easy for you to learn, too.
 
-[Language tour](https://www.dartlang.org/docs/dart-up-and-running/ch02.html)
+[Language tour](https://www.dartlang.org/guides/language/language-tour)
 : Your best introduction to the Dart language. Learn about Dart's
   features such as _optional types_, _closures_, _libraries_, _lexical scoping_,
   _top-level functions_, _named parameters_, _async / await_, and lots more.
 
-[Library tour](https://www.dartlang.org/docs/dart-up-and-running/ch03.html)
+[Library tour](https://www.dartlang.org/guides/libraries/library-tour)
 : A good overview of Dart's powerful core libraries. Learn about
   Dart's support for collections, async, math, numbers, strings, JSON, and more.
 
-[Intro to Dart for Java Developers Codelab](https://codelabs.developers.google.com/codelabs/from-java-to-dart/index.html#0">Intro to Dart for Java Developers - Codelab)
+[Intro to Dart for Java Developers Codelab](https://codelabs.developers.google.com/codelabs/from-java-to-dart/index.html?index=..%2F..%2Findex#0)
```

Comment:
Is there any way to simplify this URL? It looks dodgy.

Reply:
Just https://codelabs.developers.google.com/codelabs/from-java-to-dart works for me.